rspec-mode.el

```diff
@@ -726,7 +732,18 @@ or a cons (FILE . LINE), to run one example."
   "Compilation mode for RSpec output."
   (add-hook 'compilation-filter-hook 'rspec-colorize-compilation-buffer nil t)
   (add-hook 'compilation-finish-functions 'rspec-store-failures nil t)
-  (add-hook 'compilation-finish-functions 'rspec-handle-error nil t))
+  (add-hook 'compilation-finish-functions 'rspec-handle-error nil t)
+  (add-hook 'compilation-finish-functions 'rspec-send-notification nil t))
+
+(defun rspec-send-notification (&rest ignore)
+  (if rspec-notification-command
+      (progn
+        (rspec-store-failures)
```

Comment:
Instead of calling `rspec-store-failures` here, why not add this function to the end of `compilation-finish-functions`? Pass `t` instead of `nil` as the second-to-last `add-hook` argument.